source/eventHandler.py

```diff
@@ -20,6 +20,8 @@
 
 #Some dicts to store event counts by name and or obj
 _pendingEventCountsByName={}
+# Note that due to the fact it is hard to keep NVDAObjects remaining hashable in Python3 due to their equality method being overridden,
+# the id (address) of the NVDAObject will be used as the key in the dict, not the NVDAObject itself.
 _pendingEventCountsByObj={}

```

Comment:
I think I prefer this to be renamed to something like:
```suggestion
_pendingEventCountsByObjId={}
```